src/controllers/user.controller.js

```diff
@@ -0,0 +1,49 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService
+    .getAll()
+    .then((usrs) => usrs.map(userService.normalize));
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const user = userService.normalize(req.entry);
+
+  res.send(user);
+};
+
+const create = async (req, res) => {
+  const { name } = req.body;
+
+  const user = await userService.create(name).then(userService.normalize);
```

Comment:
Consider using `await` instead of `.then()` for consistency with the `async/await` pattern used in the function.